### ISSUE TITLE
If R is stopped mid-query then various functions stop working

### ISSUE THREAD

**Issue by tomfbush** [Tatvic/RGoogleAnalytics#16]
If I stop the output of `GetReportData()` for example because it's taking too long or I realise I have made a mistake in the query, it gives the following error:

`Error in function (type, msg, asError = TRUE)  : 
  easy handled already used in multi handle''`

Moreover this seems to affect `ValidateToken()` too.

I've just been following the tutorial at http://www.tatvic.com/blog/google-analytics-data-extraction-in-r/ :)

**Comment by jdeboer** [comment]
Yes, this is an issue is other packages too that retrieve data over an internet connection. I believe it is related to httr or rcurl. A solution would be great as this needs to be addressed in the ganalytics package too.

**Comment by tomfbush** [comment]
Httr open issue for reference/in case any solutions emerge: https://github.com/hadley/httr/issues/112

**Comment by jdeboer** [comment]
Hi @tomfbush 
Reading the news file for httr, I just noticed it mentioned a fix for the issue you describe has been made to the development version of httr available on GitHub, not yet released to CRAN. So try installing the version of httr from GitHub and see if that solves the issue for you. https://github.com/hadley/httr/blob/master/NEWS.md
